synapse/lib/rstorm.py

```diff
@@ -329,6 +332,8 @@ async def __anit__(self, rstfile):
         self.core = None
 
         self.handlers = {
+            'python': self._handlePython,
```

Comment:
I thought these were going to allow you to run arbitrary commands?

Reply:
this is what we talked about - run a python command. This fits with the docs too because when it shows the "query" in the code block, you get the actual command that needs to be executed to run the tool. We can talk about it again at next scrum if you want.

Reply:
I don't know when we had this conversation about it being arbitrary system commands. Updated now though.